tests/config/pubsub/jetstream/pubsub.yml

```
@@ -12,3 +12,5 @@ spec:
     value: config-test
   - name: flowControl
     value: true
+  - name: hearbeat
```

Should this be `heartbeat` instead?

```suggestion
  - name: heartbeat
```

---

Never mind it really is `hearbeat` - odd :D